smriprep/workflows/surfaces.py

```diff
@@ -534,7 +531,25 @@ def init_gifti_surface_wf(*, name="gifti_surface_wf"):
     fs2gii = pe.MapNode(
         fs.MRIsConvert(out_datatype="gii", to_scanner=True), iterfield="in_file", name="fs2gii"
     )
-    fix_surfs = pe.MapNode(NormalizeSurf(), iterfield="in_file", name="fix_surfs")
+    surfmorph_list = pe.Node(
+        niu.Merge(3, ravel_inputs=True),
+        name="surfmorph_list",
+        run_without_submitting=True,
+    )
+    whitesurf_list = pe.Node(
+        niu.Merge(3, ravel_inputs=True),
+        name="whitesurf_list",
+        run_without_submitting=True,
+    )
+    fscurv2funcgii = pe.MapNode(
+        fs.MRIsConvert(out_datatype="gii", to_scanner=True),
+        iterfield=["in_file", "scalarcurv_file"], name="fscurv2funcgii",
+    )
```

Comment:
We don't need to iterate if `in_file` is always the same.

```suggestion
    fscurv2funcgii = pe.MapNode(
        fs.MRIsConvert(out_datatype="gii"),
        iterfield=["scalarcurv_file"], name="fscurv2funcgii",
    )
```

Reply:
won't the in_file vary by hemisphere? I'm a little confused by the interface, I'll admit :)

Reply:
Oh right. That might be the best way to do this after all.